README.md

```diff
@@ -179,7 +179,287 @@ An example of using Drawer:
 ```
 
 ### storybook
-Check out our components at https://silinternational.github.io/ui-components/
+Try out our components at https://silinternational.github.io/ui-components/
+
+### CSS utility classes
+These are copied directly from globall.css and can be applied to all components and even components and elements from your app.
+
+.h-100 {
```

Comment:
I'm not sure that it's helpful to copy these into the README. It's asking for extra maintenance to make sure they continue to match the code.

Reply:
Good point. I have used a link instead.